Tests/UnitTests/AnyCodablePathOptionsTests.swift

```diff
@@ -829,56 +829,32 @@ class AnyCodablePathOptionsTests: XCTestCase, AnyCodableAsserts {
         """
 
         XCTExpectFailure("Validation should fail when key name is present") {
-            assertTypeMatch(expected: expected, actual: actual, pathOptions: KeyMustBeAbsent(keyNames: "key1"))
+            assertTypeMatch(expected: expected, actual: actual, pathOptions: KeyMustBeAbsent(paths: "key1"))
         }
     }
 
-    func testKeyMustBeAbsent_WithMissingKeyNames_Fails() {
-        let expected = """
-        {}
-        """
-
-        let actual = """
-        {
-            "key1": 1
-        }
-        """
-
-        XCTExpectFailure("Validation should fail when key names not provided") {
-            assertExactMatch(expected: expected, actual: actual, pathOptions: KeyMustBeAbsent(paths: nil))
-        }
-        XCTExpectFailure("Validation should fail when key names not provided") {
-            assertExactMatch(expected: expected, actual: actual, pathOptions: KeyMustBeAbsent(paths: [nil], keyNames: []))
-        }
-        XCTExpectFailure("Validation should fail when key names not provided") {
-            assertTypeMatch(expected: expected, actual: actual, pathOptions: KeyMustBeAbsent(paths: nil))
-        }
-        XCTExpectFailure("Validation should fail when key names not provided") {
-            assertTypeMatch(expected: expected, actual: actual, pathOptions: KeyMustBeAbsent(paths: [nil], keyNames: []))
-        }
-    }
-
-    func testKeyMustBeAbsent_WithInnerPath_CorrectlyFails() {
-        let expected = """
-        {}
-        """
-
-        let actual = """
-        {
-          "events": [
-            {
-              "request": {
-                "path": "something"
-              }
-            }
-          ],
-          "path": "top level"
-        }
-        """
-        XCTExpectFailure("Validation should fail when key names not provided") {
-            assertTypeMatch(expected: expected, actual: actual, pathOptions: KeyMustBeAbsent(paths: "events", keyNames: "path", scope: .subtree))
-        }
-    }
+    // TODO: Wildcard keyname/index functionality
+    //    func testKeyMustBeAbsent_WithInnerPath_CorrectlyFails() {
+    //        let expected = """
+    //        {}
+    //        """
+    //
+    //        let actual = """
+    //        {
+    //          "events": [
+    //            {
+    //              "request": {
+    //                "path": "something"
+    //              }
+    //            }
+    //          ],
+    //          "path": "top level"
+    //        }
+    //        """
+    //        XCTExpectFailure("Validation should fail when key names not provided") {
+    //            assertTypeMatch(expected: expected, actual: actual, pathOptions: KeyMustBeAbsent(paths: "events[*].request.path", scope: .subtree))
```

Comment:
Is it possible to keep this test for now using something like `events[0].request.path` while keeping the note to add wildcard functionality?